src/components/Sidebar/Sidebar.scss

```diff
@@ -0,0 +1,31 @@
+.sidebarContainer {
+  .sidebar {
+    .tabTitle {
+      cursor: pointer;
+
+      h4 {
+        font-size: 14px;
+        margin: 0;
+      }
+
+      @media screen and (max-width: 900px) {
+        .active,
+        .notActive {
+          padding: 15px;
+          display: flex;
+          // align-items: center;
+          // justify-content: center;
```

Comment:
- [ ] Kindly, remove the commented code present in this file in order to improve the repo's cleanliness. 👍